spec/controllers/catalog_controller_spec.rb

```diff
@@ -99,6 +99,21 @@
       post :create, params: { druid: druid, incoming_version: ver, incoming_size: size, storage_location: storage_location_param }
       expect(response.body).to include(druid_no_pre)
     end
+    context 'can take two forms of druid input' do
+      let(:druid) { 'druid:jj925bx9565' }
+      let(:druid_no_pre) { 'jj925bx9565' }
```

Comment:
a more commonly used name for this would be `bare_druid`  :-)   You might even consider `bare_druid` and `prefixed_druid` to be super-duper clear.   And even better:

```ruby
let(:bare_druid) {'asdfasdfsadf'}
let(:prefixed_druid {"druid:#{bare_druid}"}
```